game/scripts/vscripts/abilities/swiper/boss_swiper_reapers_rush.lua

```diff
@@ -0,0 +1,225 @@
+LinkLuaModifier("boss_swiper_reapers_rush_active", "abilities/swiper/boss_swiper_reapers_rush.lua", LUA_MODIFIER_MOTION_NONE)
+
+boss_swiper_reapers_rush = class(AbilityBaseClass)
+
+--------------------------------------------------------------------------------
+
+function boss_swiper_reapers_rush:OnAbilityPhaseStart()
+	if IsServer() then
+		local caster = self:GetCaster()
+		local width = self:GetSpecialValueFor("radius")
+		local target = self:GetCursorPosition()
+		local distance = (target - caster:GetAbsOrigin()):Length2D()
+		local castTime = self:GetCastPoint()
+		local direction = (target - caster:GetAbsOrigin()):Normalized()
+
+		DebugDrawBoxDirection(caster:GetAbsOrigin(), Vector(0,-width / 2,0), Vector(distance,width / 2,50), direction, Vector(255,0,0), 1, castTime)
+		DebugDrawCircle(target + Vector(0,0,32), Vector(255,0,0), 128, width, false, castTime + 2.0)
+	end
+	return true
+end
+
+--------------------------------------------------------------------------------
+
+function boss_swiper_reapers_rush:GetPlaybackRateOverride()
+	return 0.275
+end
+
+--------------------------------------------------------------------------------
+
+function boss_swiper_reapers_rush:OnSpellStart()
+	if IsServer() then
+		local caster = self:GetCaster()
+		local target = self:GetCursorPosition()
+		local distance = (target - caster:GetAbsOrigin()):Length2D()
+		local direction = (target - caster:GetAbsOrigin()):Normalized()
+
+		caster:Stop()
+
+		local modifier = caster:AddNewModifier(caster, self, "boss_swiper_reapers_rush_active", {})
+		local modifierTable = {}
+		modifierTable.speed = self:GetSpecialValueFor("speed")
+		modifierTable.distance = distance
+		modifier:InitRush(modifierTable)
```

Comment:
This code should be handled on the modifier itself (OnCreated) , you can send the data needed in the KV parameter